libs/partners/ai21/tests/integration_tests/test_semantic_text_splitter.py

```diff
@@ -0,0 +1,143 @@
+    "When the franchise was released internationally, the short form of the title was "
+    "used, with an acute accent (´) "
+    "over the e to aid in pronunciation.\n"
+    "Pokémon refers to both the franchise itself and the creatures within its "
+    "fictional universe.\n"
+    "As a noun, it is identical in both the singular and plural, as is every "
+    "individual species name;[10] it is "
+    'grammatically correct to say "one Pokémon" and "many Pokémon", as well '
+    'as "one Pikachu" and "many Pikachu".\n'
+    "In English, Pokémon may be pronounced either /'powkɛmon/ (poe-keh-mon) or "
+    "/'powkɪmon/ (poe-key-mon).\n"
+    "The Pokémon franchise is set in a world in which humans coexist with creatures "
+    "known as Pokémon.\n"
+    "Pokémon Red and Blue contain 151 Pokémon species, with new ones being introduced "
+    "in subsequent games; as of December 2023, 1,025 Pokémon species have been "
+    "introduced.\n[b] Most Pokémon are inspired by real-world animals;[12] for example,"
+    "Pikachu are a yellow mouse-like species[13] with lightning bolt-shaped tails[14] "
+    "that possess electrical abilities.[15]\nThe player character takes the role of a "
+    "Pokémon Trainer.\nThe Trainer has three primary goals: travel and explore the "
+    "Pokémon world; discover and catch each Pokémon species in order to complete their"
+    "Pokédex; and train a team of up to six Pokémon at a time and have them engage "
+    "in battles.\nMost Pokémon can be caught with spherical devices known as Poké "
+    "Balls.\nOnce the opposing Pokémon is sufficiently weakened, the Trainer throws "
+    "the Poké Ball against the Pokémon, which is then transformed into a form of "
+    "energy and transported into the device.\nOnce the catch is successful, "
+    "the Pokémon is tamed and is under the Trainer's command from then on.\n"
+    "If the Poké Ball is thrown again, the Pokémon re-materializes into its "
+    "original state.\nThe Trainer's Pokémon can engage in battles against opposing "
+    "Pokémon, including those in the wild or owned by other Trainers.\nBecause the "
+    "franchise is aimed at children, these battles are never presented as overtly "
+    "violent and contain no blood or gore.[I]\nPokémon never die in battle, instead "
+    "fainting upon being defeated.[20][21][22]\nAfter a Pokémon wins a battle, it "
+    "gains experience and becomes stronger.[23] After gaining a certain amount of "
+    "experience points, its level increases, as well as one or more of its "
+    "statistics.\nAs its level increases, the Pokémon can learn new offensive "
+    "and defensive moves to use in battle.[24][25] Furthermore, many species can "
+    "undergo a form of spontaneous metamorphosis called Pokémon evolution, and "
+    "transform into stronger forms.[26] Most Pokémon will evolve at a certain level, "
+    "while others evolve through different means, such as exposure to a certain "
+    "item.[27]\n"
+)
+
+
```

Comment:
You have `test_invoke` prefix to all of your tests, where in-fact you never call `invoke` method. the convention is - 
`test<func_name>__when_<use_case>__should_<do_what_is_expected>`